scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/FragmentParameterValidator.scala

```diff
@@ -55,21 +55,22 @@ object FragmentParameterValidator {
       refClazz: FragmentClazzRef,
       paramName: ParameterName,
       nodeIds: Set[String]
-  ): ValidatedNel[PartSubGraphCompilationError, Unit] =
-    valueEditor match {
-      case ValueInputWithFixedValuesProvided(_, _) =>
-        if (List(FragmentClazzRef[java.lang.Boolean], FragmentClazzRef[String]).contains(refClazz))
-          Valid(())
-        else
+  ): ValidatedNel[PartSubGraphCompilationError, Unit] = {
+    val permittedTypesForEditors = List(
```

Comment:
instead of defining it twice (here and in test), let's just define it in `object FragmentParameterValidator`